### ISSUE TITLE
TF baseline training config

### ISSUE THREAD

**Issue by Yuxinn-J** [guandeh17/Self-Forcing#36]
Hello, thanks for your great work. very intriguing!

Could you share the specific training hyperparameters for the frame-wise AR model with teacher forcing as shown in Table 2 (Right)? optimizer, learning rate and the cfg weight...?  This would be very helpful for reproducing the TF baseline. Thanks in advance :-)

**Comment by fikry102** [comment]
Any update?
Thanks

**Comment by fikry102** [comment]
<img width="1097" height="540" alt="Image" src="https://github.com/user-attachments/assets/dd8cde96-5f93-4f78-88a6-6b83c5b9a9bc" />

How to reproduce TF + DMD？ "In Teacher Forcing, the model is trained to denoise each frame conditioned on the preceding clean, ground-truth context frames."  
I see that SF+DMD use this dataset called prompts/vidprom_filtered_extended.txt. How about TF+DMD?

<img width="787" height="204" alt="Image" src="https://github.com/user-attachments/assets/0bab0255-774e-4c93-9e12-11ce6af2de23" />
